dts/arm64/nxp/nxp_imx8mm_a53.dtsi

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+#include <arm64/armv8-a.dtsi>
+#include <dt-bindings/clock/imx_ccm.h>
+#include <dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	chosen {
+		zephyr,console = &uart2;
+		zephyr,shell-uart = &uart2;
+		zephyr,sram = &sram0;
+	};
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <0>;
+		};
+
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <1>;
+		};
+	};
+
+	arch_timer: timer {
+		compatible = "arm,armv8-timer";
+		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 14 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 11 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 10 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>;
+		label = "arch_timer";
+		interrupt-parent = <&gic>;
+	};
+
+	gic: interrupt-controller@38800000 {
+		compatible = "arm,gic";
+		reg = <0x38800000 0x10000>, /* GIC Dist */
+		      <0x38880000 0xc0000>; /* GICR (RD_base + SGI_base) */
+		interrupt-controller;
+		#interrupt-cells = <4>;
+		label = "GIC";
+		status = "okay";
+	};
+
+	ana_pll: ana_pll@30360000 {
+		compatible = "nxp,imx-ana";
+		reg = <0x30360000 DT_SIZE_K(64)>;
+		label = "ANA_PLL";
+	};
+
+	ccm: ccm@30380000 {
+		compatible = "nxp,imx-ccm";
+		reg = <0x30380000 DT_SIZE_K(64)>;
+		label = "CCM";
+		#clock-cells = <3>;
+	};
+
+	uart2: serial@30890000 {
+		compatible = "nxp,imx-iuart";
+		reg = <0x30890000 DT_SIZE_K(64)>;
+		interrupts = <GIC_SPI 27 IRQ_TYPE_LEVEL 0>;
+		interrupt-names = "irq_0";
+		interrupt-parent = <&gic>;
+		clocks = <&ccm IMX_CCM_UART_CLK 0x6c 24>;
```

Comment:
The register offsets are wrong. Even in the M4 dts I did the same thing. Since the offsets were not used by the clock driver, these are irrelevant but we should fix the CCM binding to use `#clock-cells = <1>`.

This will also require changing a few dts, but that shouldn't be a big thing.

Reply:
If you think ccm binding need to change to `#clock-cells = <1>`, I could create the other PR to address it. But in this PR, I would not. I am not sure what 0x6c/24 means here, I just refer the m4 dtsi.

Reply:
Okay. Let's handle it later.